modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -290,7 +290,12 @@ export class {{classname}} extends runtime.BaseAPI {
         return new runtime.JSONApiResponse(response, (jsonValue) => jsonValue.map({{returnBaseType}}FromJSON));
         {{/isListContainer}}
         {{^isListContainer}}
+        {{#isMapContainer}}
+        return new runtime.JSONApiResponse<any>(response);
```

Comment:
Without any `transfromer` (missing second argument for the constructor), how would it process nested objects in this map?

All other types are (recursively) processed via `<someBaseType>FromJSON()>`. This processing includes mapping between raw property names and camelCase / snake_case / whatever is configued in `modelPropertyNaming`. Also this processing includes serializing / deserializing dates

Reply:
Thank you very much for the review! Indeed I tried to take the easy way out :)

I found `runtime.mapValues` which should do exactly what we need here - construct a free-form key-value object while running a transformer over the values.